package.json

```
@@ -12,12 +12,16 @@
     "pretest": "lerna run pretest",
     "test": "node scripts/test.js",
     "posttest": "lerna run posttest",
-    "test:watch": "node scripts/test.js --watch",
-    "lint": "tsc -b && yarn prettier:check && eslint --ext .js,.jsx,.ts,.tsx,.graphql .",
-    "lint:fix": "eslint --ext .js,.jsx,.ts,.tsx,.graphql . --fix; prettier --ignore-path .eslintignore --write '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "prettier:check": "prettier --ignore-path .eslintignore --check '**/*.{js,jsx,ts,tsx,graphql,md}'",
-    "depcheck": "lerna exec --stream 'yarn depcheck --ignores=\"@app/config,@app/client,tslib,webpack,babel-plugin-import,source-map-support,@graphql-codegen/*,graphql-toolkit,net,tls,dayjs\" --ignore-dirs=\".next\"'",
-    "dev": "yarn && lerna run codegen --stream && tsc -b && concurrently --kill-others --names 'TSC,WATCH,RUN,TEST' --prefix '({name})' --prefix-colors 'yellow.bold,yellow.bold,cyan.bold,greenBright.bold' 'tsc -b --watch --preserveWatchOutput --assumeChangesOnlyAffectDirectDependencies' 'lerna run --parallel watch' 'lerna run --parallel dev' 'sleep 10; yarn test:watch'",
+    "test:watch": "node scripts/test.js --watch --delay 10",
```

Move the delay back into the `dev` command below

```suggestion
    "test:watch": "node scripts/test.js --watch",
```

---

(See next comment)